properties.py

```diff
@@ -603,12 +607,12 @@ def display_driver_items(self, context):
 
     do_denoise = BoolProperty(
         name="Denoise Post-Process",
-        description="Denoise the image.  This will let set your sampling values low and get faster render times and runs denoise to remove the noise as a post process.",
+        description="Use the image denoiser to post process your render.\nThis allows you to use a lower max sample count while still getting a good image.",
```

Comment:
This is along the lines of what Lightyear was saying to me so I'll retweak this.